src/Icons/appsettings.json

```diff
@@ -10,5 +10,19 @@
     "cacheEnabled": true,
     "cacheHours": 24,
     "cacheSizeLimit": null
+  },
+  "Serilog": {
+    "Enrich": [ "FromLogContext" ],
+    "Properties": {
+      "Project": "Icons"
+    },
+    "Filter": [
+      {
+        "Name": "ByExcluding",
+        "Args": {
+          "expression": "EventId.Id = 12482444"
+        }
+      }
+    ]
```

Comment:
Not sure if we should include this logic everywhere. It feels like we should use a log level here instead, like debug? @joseph-flinn any thoughts?

If we do decide to use debug log level we can remove the `Serilog.expressions` package.

Reply:
I would tend to agree with the log level statement. While it is an interesting thought to be able to add filters to logs from the code base configuration, that should really either be handled with log levels at the application side or with the chosen central logging solution that the user/operator chooses to use.